stylus/cozy-ui/defaults.styl

```diff
@@ -32,7 +32,20 @@ textarea
 
 // UTILS
 @import '../ui-base/mixins'
+@import '../ui-base/palette'
 :global
+    .coz-error
+        color  pomegranate
+
+        &.coz-error--warning:before
```

Comment:
@m4dz Funny you asked because I tend to think that there's not enough. I may be wrong but I think when you make it too flexible, you make it too complicated to use. 
It should be easy as pie for little elements or utility classes, like "you want to display an error text in red then use this class. But if you want a big component, you'll have to extend it mate, too big for the default css". You see what I mean? I'd be happy to discuss this though, despite my firstname initials I ain't omniscient, I may miss a point.